main.go

```diff
@@ -54,7 +55,8 @@ func setupFlags(name string) (*flag.FlagSet, *options) {
 	// TODO: make this a positional arg, set a Usage func to document it
 	flags.StringVar(&opts.source, "source", ".", "package path for source structs")
 
-	// TODO: make this a positional arg, set a Usage func to document it
+	flags.StringVar(&opts.moduleWorkDir, "modworkdir", "", "module working directory to run from")
```

Comment:
Needed this so the E2E test could use it after converting `internal/e2e` into a submodule.